wasm-wrappers/js-bindings/wasm_test.js

```diff
@@ -776,6 +779,43 @@ export async function run_test() {
     );
     console.log("htlc with tokens encoding ok");
 
+    encode_create_order_output(
```

Comment:
add expected bytes for the encoding like the other tests so we can keep track if anything brakes in the future.